### PR TITLE
modules/SceMotion: stub GetBasicOrientation.

### PR DESCRIPTION
**Pull request by Zangetsu38**
fix orientation change alone on pinball aracade.

### CODE REVIEW

File: vita3k/modules/SceMotion/SceMotion.cpp [hunk 1]
```diff
@@ -21,8 +21,16 @@ EXPORT(int, sceMotionGetAngleThreshold) {
...
+EXPORT(int, sceMotionGetBasicOrientation, SceFVector3 *basicOrientation) {
+    if (!basicOrientation)
+        return RET_ERROR(SCE_MOTION_ERROR_NULL_PARAMETER);
+
+    STUBBED("set default value");
+    basicOrientation->x = 0;
+    basicOrientation->y = 0;
+    basicOrientation->z = 1;
+
+    return SCE_OK;
```
Comment (scribam) [inline vita3k/modules/SceMotion/SceMotion.cpp:33]:
```suggestion
    return 0;
```

File: vita3k/modules/SceMotion/SceMotion.h [hunk 2]
```diff
@@ -19,6 +19,21 @@
 
 #include <module/module.h>
 
+enum SceMotionErrorCode {
+    SCE_OK = 0,
```
Comment (scribam) [inline vita3k/modules/SceMotion/SceMotion.h:23]: Remove this line

**Review by scribam — CHANGES_REQUESTED** [review]
The constant `SCE_OK` is never used, I think `0` is preferred.

File: vita3k/modules/SceMotion/SceMotion.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/modules/SceMotion/SceMotion.cpp]: is done for both

**Review by scribam — APPROVED** [review]

**Pull request merged**